app/jobs/hackney/income/jobs/save_and_send_letter_job.rb

```diff
@@ -14,6 +14,7 @@ def perform(letter_html:, bucket_name:, filename:, document_id:)
           @document_id = document_id
           binary_letter_content = generate_pdf_binary(letter_html, document.uuid)
 
+          Rails.logger.info "uploading #{filename} to #{bucket_name}"
```

Comment:
Maybe also show the doc id for if something goes wrong and we need to hunt down that upload
```suggestion
          Rails.logger.info "uploading document: #{document_id} - #{filename} to #{bucket_name}"
```

Reply:
:+1: